app/models/mixins/retirement_mixin.rb

```diff
@@ -1,9 +1,10 @@
 module RetirementMixin
   extend ActiveSupport::Concern
 
-  RETIRED  = 'retired'
-  RETIRING = 'retiring'
-  ERROR_RETIRING = 'error'
+  RETIRED  = 'retired'.freeze
+  RETIRING = 'retiring'.freeze
+  ERROR_RETIRING = 'error'.freeze
+  RETIREMENT_INITIALIZING = 'initializing'.freeze
```

Comment:
Do these constants get used outside of this module?  If not it would be nice to standardize on naming and sort them.  I would suggest something like:
```ruby
RETIREMENT_ERROR = 'error'.freeze
RETIREMENT_INITIALIZING = 'initializing'.freeze
RETIREMENT_RETIRED  = 'retired'.freeze
RETIREMENT_RETIRING = 'retiring'.freeze
```